pkg/controller/component/taskrun.go

```diff
@@ -2,12 +2,12 @@ package component
 
 import (
 	"fmt"
-	"github.com/knative/pkg/apis"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	"halkyon.io/api/component/v1beta1"
 	"halkyon.io/operator/pkg/controller/framework"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"knative.dev/pkg/apis"
```

Comment:
Where is this used?

Reply:
`knative.dev/pkg/apis` is used at the line 103 of this file to check the status.